.gitignore

```diff
@@ -40,6 +41,8 @@ cpp/unreal_projects/SpearSim/.vsconfig
 cpp/unreal_projects/SpearSim/.ignore
 cpp/unreal_projects/SpearSim/.kdev4
 
+.DS_Store
```

Comment:
Is this needed? If this was a result of some early prototyping, and we don't plan to generate any images in our base folder on an ongoing basis, this arguably isn't needed here.